app.js

```diff
@@ -1,11 +1,13 @@
 const express = require('express');
 const app = express();
-const { retreiveTopics } = require('./controllers/controller');
-
-app.get('/api/topics', retreiveTopics);
-
+const endpointsData = require('./endpoints.json');
+const { retrieveTopics, retrieveArticleById } = require('./controllers/controller');
 
+app.get('/api/topics', retrieveTopics);
 
+app.get('/api', (req, res) => {
+    res.status(200).send(endpointsData);
```

Comment:
Any data sent from a controller needs to be sent back on a meaningful key ie. .send({endpointsData})